### DISCUSSION TITLE
Cannot delete user account

### DISCUSSION THREAD

**Discussion by dominiksramko** [vercel/community#5341] [Account Issue]
### Question

Browsing [Account Settings](https://vercel.com/account) shows 404 when logged in. There's no option to delete the entire account. Tried on different computers, same result. How do I proceed?
Is there an email address I can use to exercise my right to be forgotten?

![error](https://github.com/vercel/community/assets/5091255/0131235f-a7d0-4471-90f6-267945e1f3df)

Thank you

**Comment by krauzer** [comment]
Have the same issue: #5355

**Comment by amyegan** [comment]
Hi @dominiksramko. Are you still running into this error?

**Comment by dominiksramko** [comment]
@amyegan Yes, the screen is still showing the same error. I don't understand why it says the name of the team in the top left corner as it has been deleted beforehand. Though I am sure that's the core of the issue.

**Reply by amyegan** [reply]
Does the dropdown at the top only show the deleted team, or do you have another option there? 

If you share the name of the team I can look into it for you 🙏

**Reply by dominiksramko** [reply]
No, there was only one team associated with the account. The name of the team was "NW Developer".
Edit: The dropdown doesn't actually show anything, but the deleted team shows up as the currently selected one.

**Reply by amyegan** [reply]
Thanks for sharing that. I don't see a recently deleted Pro team by that name associated with your GitHub username. Are you logged into the Vercel account associated with this GitHub username, or is this problem happening on a different Vercel account?

**Reply by dominiksramko** [reply]
It's associated with an e-mail account tied to my company's boss. He signed up using e-mail directly. I am just the middleman.  I didn't feel comfortable sharing that here directly, hence the blurring.

**Reply by dominiksramko** [reply]
I think it would be easiest if you could just provide us with an e-mail address we can contact to issue a right to be forgotten request directly. Would that be possible?

**Reply by amyegan** [reply]
Of course! EEA and UK customers can submit a request by emailing us at [privacy@vercel.com](mailto:privacy@vercel.com) with the term "DSR" as the subject line. 

You can find more information in our Privacy Policy: https://vercel.com/legal/privacy-policy#dsr
